### DISCUSSION TITLE
Wanna know something about push users

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#17290] [Ideas]
![Screenshot_20230125-083637_Brave](https://user-images.githubusercontent.com/62275326/214468242-d90d5271-5ae0-49de-aeb1-420e984a0906.png)

Build link: https://cirrus-ci.com/build/5727298691792896 
Why am I getting needs approval? Although I am new push user. Is it because my previous build of banana was 10h+ in queue and got pr blocked?

**Comment by Apon77** [comment]
No Idea about this. Only cirrus knows
